src/screens/FeedScreen/RenderList.js

```diff
@@ -171,7 +271,8 @@ const RenderListFeed = (props) => {
             statusVote={voteStatus}
             showScoreButton={showScoreButton}
             onPressScore={() => showScoreAlertDialog(item)}
-            isSelf={item.anonimity ? false : selfUserId === item.actor.id}
+            isSelf={isSelf}
+            isShowDM
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `isSelf` prop has been replaced with a new value and `isShowDM` has been added. Ensure that these props are being used correctly in the `Footer` component.

```diff
- isSelf={item.anonimity ? false : selfUserId === item.actor.id}

+ isSelf={isSelf}
+ isShowDM
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->